src/components/TransactionsTable.tsx

```diff
@@ -43,8 +43,12 @@ export default function TransactionsTable({
         },
         order: {
           fk_transaction: {
-            date: 'ASC',
+            date: 'DESC',
           },
+          // This is so debit is always before credit
+          // so we avoid negative amounts when display
+          // partial totals
```

Comment:
this fixes all accounts but INCOME I think but I haven't seen a case yet. Will fix when it appers.